test/loop-protect.test.js

```diff
@@ -89,7 +89,7 @@ describe('loop', function() {
   });
 
   it('console error when passing string', () => {
-    const code = `var i = 0; while (true) i++; return true`;
+    const code = `var i = 0; while (true) i++; return true;`;
```

Comment:
```suggestion
    const code = `var i = 0; while (true) i++; return true`;
```
Not that it matters, but this change seems unnecessary.